packages/components/src/index.ts

```diff
@@ -7,6 +7,7 @@ export * from './support/ThemeProvider';
 
 export * from './components/Badge/Badge';
 export * from './components/buttons/Button/Button';
+export * from './components/Divider/Divider';
```

Comment:
This component is going to be [merged separately](https://github.com/trezor/trezor-suite/pull/11252/files), because we use it in [different PR](https://github.com/trezor/trezor-suite/pull/11187/files). 
Please delete its import and implementation.